platform/core/src/classes/metadata/StudyMetadata.js

```diff
@@ -626,6 +692,10 @@ const makeDisplaySet = (series, instances) => {
 
   imageSet.isReconstructable = isReconstructable.value;
 
+  if (shallSort && imageSet.isReconstructable) {
```

Comment:
shallSort sounds odd

Reply:
We sort because we need the correct order for 2D MPR? Why not sort even if its not reconstructable?

Reply:
@JamesAPetts this is for you my friend.

Reply:
shallSort was already present above, always set `true`, its already in master. We could get rid of it now, it was likely placeholder.

> Why not sort even if its not reconstructable?

It only makes sense to sort by `ImagePositionPatient `for volumes, you don't want to attempt to sort an SM or an SR. The regular sort above (line 676) sorts by instance number, which is the best thing to do for an unreconstructable set of images.

One could refactor the `isReconstructable` logic into a generic "sort" that has two paths though 🤷‍♂ .

Reply:
Something to be done here? @dannyrb @JamesAPetts